meta-ov/recipes-apps/ovmenu-ng-skripts/files/system-info.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+#
+# system-info.sh
+# System info script gives an overview which packages and which versions are instaled
+#
+# Created by Blaubart,        2022-02-08
+#
+
+# collect info of system and more installed packages
+IMAGE_VERSION=$(cat /etc/os-release | grep VERSION_ID | cut -d '=' -f 2)
+KERNEL_VERSION=$(uname -r)
+XCSOAR_MAPS_VERSION=$(opkg list-installed xcsoar-maps* | cut -d '-' -f 4)
+XCSOAR_MENU=$(opkg list-installed xcsoar-menu* | cut -d '-' -f 3)
+IP_ETH0=$(ip route | grep eth0 | head -n 1 | cut -d ' ' -f 8)
+MAC=`ip li|grep -A 1 eth0|tail -n 1|cut -d ' ' -f 6`
+HOSTNAME=$(cat /etc/hostname)
+IP_WLAN=$(ip route | grep wlan0 | head -n 1| cut -d ' ' -f 8)
+I2C_TOOLS=$(opkg list-installed i2c-tools | cut -d '-' -f 3)
+E2FSPROGS=$(opkg list-installed e2fsprogs | cut -d '-' -f 2)
+USB_MODESWITCH=$(opkg list-installed usb-modeswitch | cut -d '-' -f 3)
+
+# collect status of SSH, variod and sensord
+if  /bin/systemctl --quiet is-enabled dropbear.socket
+then 
+	SSH_STATUS=enabled
+else 
+	SSH_STATUS=disabled
+fi
+
```

Comment:
This commit does three things:
- move code to a new script (why?)
- reformat the code
- add new features

Doing 3 distinct things in 1 commit obscures the code changes, because one cannot see what was changed; the commitdiff is "remove 20 lines here, add 100 lines over there", but no diff possible.